kse/src/org/kse/gui/crypto/customextkeyusage/resources_de.properties

```diff
@@ -0,0 +1,13 @@
+CustomExtKeyUsageTableHeadRend.ObjectId.tooltip=
```

Comment:
It is better to not add any keys for languages that you cannot translate yourself to. Then the english phrases are shown instead of nothing. Translation tools like ResourceBundle Editor make it easy to find missing texts.